components/brave_rewards/browser/net/network_delegate_helper.cc

```diff
@@ -12,7 +12,7 @@
 #include "brave/components/brave_rewards/browser/rewards_service.h"
 #include "brave/components/brave_rewards/browser/rewards_service_factory.h"
 #include "chrome/browser/profiles/profile.h"
-#include "chrome/browser/sessions/session_tab_helper.h"
+#include "components/sessions/content/session_tab_helper.h"
```

Comment:
no dep needed for this?

Reply:
It's already there: https://github.com/brave/brave-core/blob/master/components/brave_rewards/browser/BUILD.gn#L65